ledger-core/inc/core/operation/OperationDatabaseHelper.inl

```diff
@@ -0,0 +1,193 @@
+/*
```

Comment:
I suggest you keep `.cpp` as file name. It contains implementation and I’m not sure we really care about naming files `.inl` just for when we have templated function implementations. Ping @LedgerHQ/core-library for other hindsight.